proof/refine/AARCH64/Init_R.thy

```diff
@@ -29,22 +29,19 @@ context begin interpretation Arch . (*FIXME: arch-split*)
   system.
 *)
 
-definition zeroed_arch_abstract_state ::
-  arch_state
-  where
+definition zeroed_arch_abstract_state :: arch_state where
```

Comment:
checking: you did the update to new style on other arches before as well right?

Reply:
Other arches aren't done yet, I'll update them when copying across the changes from this PR.